pkg/manager/manager_tasks.go

```diff
@@ -86,6 +78,55 @@ func (t *TaskStatus) updated() {
 	t.LastUpdate = time.Now()
 }
 
+func (s *singleton) neededScan() (total *int, newFiles *int) {
+	const timeout = 90 * time.Second
+
+	// create a control channel through which to signal the counting loop when the timeout is reached
+	chTimeout := time.After(timeout)
+
+	logger.Infof("Counting files to scan...")
+
+	t := 0
+	n := 0
+
+	timeoutErr := errors.New("timed out")
+
```

Comment:
`stoppingErr := errors.New("stopping")`
we could use a stopping error also like the scan process for more verbosity

Reply:
The behaviour is the same regardless of time out or stopping, and the text itself is not displayed. Logging for stopping is handled in the calling method.